src/validation.cpp

```diff
@@ -3392,6 +3392,9 @@ bool ProcessNewBlockHeaders(const std::vector<CBlockHeader>& headers, CValidatio
         }
     }
     NotifyHeaderTip();
+    if (IsInitialBlockDownload() && ppindex && *ppindex)
+        LogPrintf("Synchronizing blockheaders, height: %d (~%.2f%%)\n",
```

Comment:
if its possible, i wanna short this message. like "Sync headers, height=2000 (0.05%)"

Reply:
i would to say its downloading rather than sync. because not yet synced actually?